src/resources/nfts/types.ts

```diff
@@ -26,12 +19,12 @@ export type NFTMarketplace = {
 export type NFTFloorPrice = {
   marketplaceId: NFTMarketplaceId;
   value: number;
-  paymentToken: NFTPaymentToken;
+  paymentToken: Asset;
```

Comment:
unrelated but realized `NFTPaymentToken` was essentially a duplicate of `Asset` so wanted to replace it

Reply:
_Should_ they be the same? Should the be allowed to evolve independently?